transmitter.go

```diff
@@ -10,88 +10,38 @@ import (
 	"github.com/linxGnu/gosmpp/pdu"
 )
 
-const (
-	// EnquireLinkIntervalMinimum represents minimum interval for enquire link.
-	EnquireLinkIntervalMinimum = 20 * time.Second
-)
-
 var (
 	// ErrTransmitterClosing indicates transmitter is closing. Can not send any PDU.
 	ErrTransmitterClosing = fmt.Errorf("Transmitter is closing. Can not send PDU to SMSC")
```

Comment:
Maybe this error should be renamed to something more generic ?
```
	// ErrConnectionClosing indicates connection is closing, can not send any PDU.
	ErrConnectionClosing = fmt.Errorf("connection is closing, can not send PDU to SMSC")
```
Just an hint, my IDE suggests me that error string should not be capitalized or end with punctuation.

Reply:
Yeah, it's great suggestion. I will address this